es-vue-base/tests/EsButton.spec.js

```diff
@@ -1,6 +1,6 @@
 import { mount } from '@vue/test-utils';
-import EsButton from '../src/lib-components/es-button.vue';
```

Comment:
Why remove relative imports?

Reply:
Consistency and ultimately when/if the file moves the imports don't break.